index.md

```diff
@@ -39,46 +37,27 @@ It is often good as a fascist to play straightforward, logically and for the mos
 
 Feigning ignorance is often a good way to cover up your manipulative actions. As strategies are formulated and the meta evolves, it's often easiest to claim “Ah! I didn’t realise that was a thing!” to cover up something that you may have done that seems out of the ordinary.
 
-Sometimes it works for fascists to create a double dispute. For example, when an investigation power to be opened,  fascist president forces chancellor to play a fascist policy and then claims, he gave a choice. When getting investigation power, investigate a liberal and claim he is a fascist. Since three people are in dispute, it is more difficult to exclude them from the game and players would be forced to take sides.
-
-Playing liberal policies is often beneficial. It burns through liberal policies (especially if you can discard one or two while doing so), meaning it is more likely that liberal governments will be forced to enact fascist policies. This increases trust for you and simultaneous increases distrust for later (well-meaning liberal) governments.
 
 ### Hitler
 
-This section is still to be completed.
+Hitler has the most sensitive role, as they can easily win or lose the game for the Fascists. A good Hitler player can worm their way into the Liberal block and get themselves nominated while still playing Fascist cards when they think they can get away with it. A poor one may get discovered and make themselves a prime target for execution powers, undermining any results the other Fascists have been achieving.
 
---------------
+The most basic Hitler strategy is to just play as a straight liberal. Try to make all the same decisions you would make as a Liberal player; playing Liberal policies, discovering Fascists, voting conservatively. If you can do this well, you should have a reasonable amount of trust from the Liberal players to get you voted in as chancellor when it matters. Of course, with this strategy you are relying on the regular Fascists to enact the policies you need to win the game, but a straight Liberal Hitler play is a good idea if you aren't comfortable trying more advanced tactics.
 
-## Game phases
-### Setup
-When examining your envelope, be sure to examine both your party membership and role cards **even if you are a liberal**. Everyone should look at both their cards otherwise you can tell who is a fascist by whoever looked at both their cards. By looking at both cards, the game starts fairly for both teams and this cannot be used against anyone throughout. This is also important for all liberals to do since if a liberal does look at both cards you may be inclined to judge them as fascist for doing so.
+When you are more familiar with the game, you can try selectively discarding Liberal policies when you can plausibly deny doing so. However, try not to get in disputes with other players, as you could quickly find yourself cut out of the game with no chance of being elected for that early victory.
```

Comment:
Ignore, I read it as "cut of" instead of "cut out of". Cut out of is perfectly fine.